# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pam/fix-empty-libcrypt.patch
# Copyright (C) 2009 - 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2010-11-25 12:31:42.138696299 +0100
+++ ./configure.in	2010-11-25 12:32:38.346698690 +0100
@@ -397,6 +397,9 @@
 
 BACKUP_LIBS=$LIBS
 AC_SEARCH_LIBS([crypt],[$crypt_libs], LIBCRYPT="-l$ac_lib", LIBCRYPT="")
+if test "x$LIBCRYPT" = "x-l"; then
+	LIBCRYPT=-lcrypt
+fi
 AC_CHECK_FUNCS(crypt_r crypt_gensalt_r)
 LIBS=$BACKUP_LIBS
 AC_SUBST(LIBCRYPT)
